pubnub-kotlin/pubnub-kotlin-api/src/appleMain/kotlin/com/pubnub/api/endpoints/presence/HereNow.ios.kt

```diff
@@ -27,18 +27,24 @@ class HereNowImpl(
     private val channels: List<String>,
     private val channelGroups: List<String>,
     private val includeState: Boolean,
-    private val includeUUIDs: Boolean
+    private val includeUUIDs: Boolean,
+    private val limit: Int = 1000,
+    private val offset: Int? = null,
 ) : HereNow {
     override fun async(callback: Consumer<Result<PNHereNowResult>>) {
         pubnub.hereNowWithChannels(
             channels = channels,
             channelGroups = channelGroups,
             includeState = includeState,
             includeUUIDs = includeUUIDs,
+            // todo pass limit and offset once available
```

Comment:
What does it mean "once available"? I believe so far we can accept parameters for users and only `nextOffset` still require clarification.

Reply:
Once new version of Swift Sdk is released with limit and offset in HereNow I can include it into this project and modify HereNow KMP function. Same for JS. Now I only include changes for Kotlin. KMP version of functions are used in KMP Chat SDK to produce JS, Kotlin, Swift chat SDKs